libs/wingc/src/jsify.rs

```diff
@@ -145,6 +145,33 @@ impl JSifier {
 			}
 		}
 	}
+	// note: Globals are emitted here and wrapped in "{{ ... }}" blocks. Wrapping makes these emissions, actual
+	// statements and not expressions. this makes the runtime panic if these are used in place of expressions.
+	fn jsify_global_utility_function(&self, args: &ArgList, function_type: UtilityFunctionKind) -> String {
+		match function_type {
+			UtilityFunctionKind::Assert => {
+				return format!(
+					"{{((cond) => {{if (!cond) throw new Error(`preflight assertion failed: '{0}'`)}})({0})}}",
+					self.jsify_arg_list(args, None, None, false)
+				);
+			}
+			UtilityFunctionKind::Print => {
+				return format!("{{console.log({})}}", self.jsify_arg_list(args, None, None, false));
+			}
+			UtilityFunctionKind::Throw => {
+				return format!(
+					"{{((msg) => {{throw new Error(msg)}})({})}}",
```

Comment:
Why are we wrapping in a function?

(Add comment explaining)

Reply:
https://github.com/winglang/wing/pull/1030#discussion_r1061166157

Reply:
Not exactly sure what's the answer here...

Reply:
Oh this one got confused with the comment below it. The reason for this one is just to stay consistent with the other ones.